hoomd_organics/base/simulation.py

```diff
@@ -95,6 +97,7 @@ def __init__(
         self._create_state(self.initial_state)
         # Add a gsd and thermo props logger to sim operations
         self._add_hoomd_writers()
+        self._thermostat = HOOMDThermostats.BUSSI
```

Comment:
I think we should use MTTK as the default. I'm pressure that is what hoomd 3 used before adding these other ones to choose from.

Reply:
Also, should we move this into the parameters and have `self._thermostat = thermostat` ?